css/style.css

```diff
@@ -1,4 +1,546 @@
+@charset "UTF-8";
+
+/*Обнуление*/ 
+
 @import url(../fonts/font.css);
+
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+*,
+*::before,
+*::after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+:focus,
+:active {
+  outline: none;
+}
+
+a:focus,
+a:active{
+  outline: none;
+}
+
+nav,
+footer,
+header,
+aside {
+  display: block;
+}
+
+html,
 body {
+  height: 100%;
+  width: 100%;
+  font-size: 100%;
+  line-height: 1;
+  font-size: 14px;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+input,
+button,
+textarea {
+  font-family: inherit;
+}
+
+input::-ms-clear {
+  display: none;
+}
+
+button {
+  cursor: pointer;
+}
+
+button::-moz-focus-inner {
+  padding: 0;
+  border: 0;
+}
+
+a,
+a:visited {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+ul li {
+  list-style: none;
+}
+
+img {
+  vertical-align: top;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+/*--------------------*/ 
+
+body {
+  margin: 0 auto;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.header {
+  padding: 24px 0 0 0;
+}
+
+.header__container {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  max-width: 1338px;
+  margin: 0 auto;
+}
+
+.header__logo {
+  padding: 0;
+}
+
+.header__menu {
+  width: 490px;
+}
+
+.menu__row {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  flex-wrap: nowrap;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.menu__item {
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+  color: #000;
+  cursor: pointer;
+  padding: 0 6px 0;
+}
+
+.menu__row .menu__item:first-child{
+  border-bottom: 2px solid #e51937;
+  color: #e51937;
+  display: block;
+}
+
+.menu__item:hover {
+  color: #e51937;
+  -webkit-transition: .5s ease-in-out;
+  -o-transition: .5s ease-in-out;
+  transition: .5s ease-in-out;
+}
+
+.header__right_row {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  flex: 0 1 203px;
+  justify-content: space-between;
+  padding: 0 15px 0;
+}
+
+.header__right_row .menu__item {
```

Comment:
Codacy found an issue: [Expected selector ".header__right_row .menu__item" to come before selector ".menu__row .menu__item:first-child" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5803911)